client/packages/invoices/src/Prescriptions/api/hooks/utils.ts

```diff
@@ -32,16 +32,17 @@ export const createInputObject = (
   line: DraftStockOutLine,
   type: 'insert' | 'update' | 'delete'
 ) => {
-  const { id, numberOfPacks, stockLine, invoiceId, note } = line;
+  const { id, numberOfPacks, prescribedQuantity, stockLine, invoiceId, note } =
+    line;
 
   const stockLineId = stockLine?.id ?? '';
-  const output = { id, numberOfPacks, stockLineId, note };
+  const output = { id, numberOfPacks, stockLineId, note, prescribedQuantity };
 
   switch (type) {
     case 'delete':
       return { id };
     case 'update':
-      return output;
+      return { ...output };
```

Comment:
I don't think you need the ...spread here, do you?

Reply:
Woops, forgot to undo this one